src/Api/Controllers/OrganizationUsersController.cs

```diff
@@ -375,5 +376,71 @@ public async Task<ListResponseModel<OrganizationUserBulkResponseModel>> BulkDele
             return new ListResponseModel<OrganizationUserBulkResponseModel>(result.Select(r =>
                 new OrganizationUserBulkResponseModel(r.Item1.Id, r.Item2)));
         }
+
+        [HttpPatch("{id}/disable")]
+        [HttpPut("{id}/disable")]
+        public async Task Disable(string orgId, string id)
```

Comment:
Use `Guid` instead of `string` and for all other parameters.

Reply:
@justindbaur , is this a desire to break the current pattern present throughout the rest of this controller class and to establish a new pattern? I just want to be sure as I understand the value of type casting during parameter binding for controllers, however we're not doing this anywhere else it seems.

From memory, you would also need to ensure the `Guid` is nullable, `Guid?` and/or assign a default value as if a `PATH` is provided that maps through the router but is missing the parameter otherwise, you get some not-so-friendly error messages on path binding which aren't apparently obvious. So there are arguments for both ways, just trying to understand the direction we should be taking.